llms/errors.go

```diff
@@ -0,0 +1,11 @@
+package llms
+
+import "errors"
+
+var (
+	// ErrEmptyResponse is thrown when an LLM returns an empty response.
```

Comment:
"thrown" isn't exactly the right language in Go,

Reply:
@tmc Fair. I'll change it to "is returned"